datalab/datalab_session/data_operations/fits_output_handler.py

```diff
@@ -0,0 +1,62 @@
+import tempfile
+import numpy as np
+from astropy.io import fits
+
+from datalab.datalab_session.file_utils import create_jpgs
+from datalab.datalab_session.s3_utils import save_fits_and_thumbnails
+
+
+class FITSOutputHandler():
+  """A class to handle FITS output files and create jpgs.
+  
+  Class handles the creation of Datalab output for developers.
+  The class inits with a cache_key and data, and creates a FITS file with the data.
+  The FITS file is then saved to the cache and the large and small jpgs are created.
+
+  Attributes:
+    dlab_id (str): The cache key for the FITS file.
+    primary_hdu (fits.PrimaryHDU): The primary HDU for the FITS file.
+    image_hdu (fits.ImageHDU): The image HDU for the FITS file.
+    data (np.array): The data for the image HDU.
+  """
+    
+  def __init__(self, cache_key: str, data: np.array, comment: str=None) -> None:
+      """Inits FITSOutputHandler with cache_key and data.
+      
+      Args:
+        cache_key (str): The cache key for the FITS file, used as an ID when stored in S3.
+        data (np.array): The data that will create the image HDU.
+        comment (str): Optionally add a comment to add to the FITS file.
+      """
+      self.dlab_id = cache_key
```

Comment:
This variable name is unnecessarily terse. `dlab_id` isn't necessarily meaningful or descriptive.